### PR TITLE
Consolidate set functions into their counterparts

### PR DESCRIPTION
**Pull request by smalyala**
Updates only the code, not the docstrings

### CODE REVIEW

File: src/python_pachyderm/mixin/pps.py [hunk 1]
```diff
@@ -134,7 +118,7 @@ def delete_job(self, pipeline_name, job_id):
             ),
         )
 
-    def stop_job(self, pipeline_name, job_id, reason=""):
+    def stop_job(self, job_id, pipeline_name, reason=""):
```
Comment (albscui) [inline src/python_pachyderm/mixin/pps.py:121]: Update the params order as well

File: src/python_pachyderm/mixin/pps.py [hunk 2]
```diff
@@ -56,68 +51,57 @@ def list_job(
...
+                jqFilter=jqFilter,
+            )
+        else:
+            return self._req(
+                Service.PPS,
+                "ListJobSet",
+                details=details,
+            )
+
+    def delete_job(self, job_id, pipeline_name):
```
Comment (albscui) [inline src/python_pachyderm/mixin/pps.py:104]: Update docstring's param order

**Review by albscui — COMMENTED** [review]
Fastt! Just some minor comments about docstring's param order.

File: src/python_pachyderm/mixin/pfs.py [hunk 3]
```diff
@@ -257,46 +257,47 @@ def commit(self, repo_name, branch, parent=None, description=None):
...
+                [
+                    self._req(
+                        Service.PFS,
+                        "InspectCommit",
+                        commit=commit_from(commit),
+                        wait=commit_state,
+                    )
+                ]
+            )
+        else:
```
[the remark this answers is not in the document]
Comment (smalyala) [reply src/python_pachyderm/mixin/pfs.py]: added

File: src/python_pachyderm/mixin/pps.py (hunk 1 above)
Comment (smalyala) [reply src/python_pachyderm/mixin/pps.py:121]: done

File: src/python_pachyderm/mixin/pps.py (hunk 2 above)
Comment (smalyala) [reply src/python_pachyderm/mixin/pps.py:104]: done

File: src/python_pachyderm/mixin/pfs.py [hunk 4]
```diff
@@ -287,13 +287,14 @@ def inspect_commit(
...
             )
-        else:
+        elif uuid_re.match(commit):
             return self._req(
                 Service.PFS,
                 "InspectCommitSet",
                 commit_set=pfs_proto.CommitSet(id=commit),
                 wait=commit_state == pfs_proto.CommitState.FINISHED,
             )
+        return None
```
Comment (albscui) [inline src/python_pachyderm/mixin/pfs.py:297]: You don't really need this, since Python automatically returns None

Comment (albscui) [reply src/python_pachyderm/mixin/pfs.py:297]: We might actually want to raise an exception in this case

Comment (smalyala) [reply src/python_pachyderm/mixin/pfs.py]: added

**Review by albscui — APPROVED** [review]
LGTM!

**Pull request merged**